apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -317,8 +317,8 @@ private function update_pattern_block_types() {
 			if ( $post_content_offset !== false ) {
 				unregister_block_pattern( $pattern['name'] );
 
-				$pattern['blockTypes'] = array_splice( $pattern['blockTypes'], $post_content_offset, 1 );
-				$pattern_name          = $pattern['name'];
+				array_splice( $pattern['blockTypes'], $post_content_offset, 1 );
```

Comment:
The `array_splice` returns an array consisting of the extracted elements... I'm unsure why it worked before 🙈